astropy/cosmology/_src/core.py

```diff
@@ -552,7 +557,7 @@ def nonflat(self: _FlatCosmoT) -> _CosmoT:
         """Return the equivalent non-flat-class instance of this cosmology."""
 
     def clone(
-        self, *, meta: Mapping | None = None, to_nonflat: bool = False, **kwargs
+        self, *, meta: Mapping | None = None, to_nonflat: bool = False, **kwargs: Any
```

Comment:
Add the parameters to `Mapping`. Even if you have to use `Any` for the value type, writing our the key type is still useful.